articles/getting-started-with-browser-sync/index.md

```diff
@@ -0,0 +1,106 @@
+ 
+>Browser-sync is awesomely fast, with incredible features for your use. Browser-sync allows you to also check your work preview simultaneously on different devices, provided you are connected on the same network.
+ 
+![Internal-UI-Interface](/engineering-education/getting-started-with-browser-sync/browsersync-dashboard.png)
+ 
+In the above image, you will see different features, these are the awesome things browser-sync provides you. Install on your computer and enjoy a seamless watch over your files. It also records your test URLs so you can push them back out to all devices with a single click.
+I will give the steps to install browser-sync for Linux. Other OS installation can be seen in their documentation [here](https://www.browsersync.io/docs)
+ 
+### Linux Installation
+ 
+The first step is to install Node.js on your computer
+```
+sudo apt install nodejs
+ 
+```
+After the installation, check the version of the installed node using the command below
+ 
+```
+node -v
+```
+ 
+Once you have successfully installed nodejs on your computer, let us get you started on how to install browser-sync.
+ 
+```
+npm install -g browser-sync
+```
+The above command will get browser-sync installed on your computer. After that, change the directory to where your project file is located. Looking at the image below, the project file is located on my Desktop. So, I do a “cd Desktop/Vue project” in my terminal. This takes me to the Desktop directory of my computer and inside the Vue project folder.
+ 
+![Change-directory](/engineering-education/getting-started-with-browser-sync/cd.png)
+ 
+The next thing to do is run the command below:
+ 
+```
+browser-sync start --server --directory --files "*"
+```
+The above command calls for browser-sync and request for a server start, then the --directory --files tell the server to watch over the files in the directory. Including the “*” means, watch over any changes I make into any file in this directory/folder.
+ 
+Using Browser-sync for dynamic sites when you are already running a local server (i.e Xampp) with PHP or a similar scripting language, the process is very similar to server mode although will require to run BrowserSync in proxy mode. This can be done using the command below:
+ 
+```
+browser-sync start --proxy "localhost/hotel" --files "*"
+```
+*Where localhost/hotel is the URL of your local server.*
+ 
+Before going to the next step, I will discuss briefly on ***Server & Proxy Server***
+ 
+### Server & Proxy Server
+_A server_ is a software or hardware device that accepts and responds to requests made over a network.
+This simply put, since your project is yet to be hosted on a live server(internet). On the Internet, the term "server" commonly refers to the system that receives requests for web files or data and sends those files back to the client (your computer).
```

Comment:
"On the Internet, the term "server" commonly refers to the system that receives requests for web files or data and sends those files back to the client (your computer)." move this before the previous sentence.